packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -549,12 +597,7 @@ const selfBehavior = {
       // TODO `await` is allowed until the above ordering is fixed
       // eslint-disable-next-line @jessie.js/no-nested-await
       const vaultKit = await vault.initVaultKit(seat);
-      state.totalCollateral = AmountMath.add(
-        state.totalCollateral,
-        vaultKit.vault.getCollateralAmount(),
-      );
```

Comment:
this didn't previously account for changes in collateral after a vault was created. it's handled now in `updateVaultAccounting`